webpack/stories/docs/internationalization.md

```diff
@@ -1,14 +1,44 @@
 # Translations
 
-All strings need to be translated. We use gettext's underscore `__()` function as you know it from our asset pipeline code.
+All strings need to be translated.
 
-Note that for the time being the underscore function is available in the global `window` object and translations are handled outside of our webpack-processed code and there's no need to import it. This is most likely subject to change in future.
+**Please note that the global underscore, sprintf and ngettext are deprecated**.
```

Comment:
I would suggest that you leave this section towards the end (assuming this is going to stay as instructions for new developers too).

Reply:
+1 it makes sense to move it lower.